_posts/2021-09-08-notes-from-community-call-2-september-2021.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Notes from the Standard for Public Code community call - 2 September 2021"
+date: 2021-09-08
+author: Jan Ainali
+type: blogpost
+excerpt: Increasing nuance in the requirement on use of English 
+categories:
+  - Community call
+---
+
+# Notes from the Standard for Public Code community call - 2 September 2021
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/team/jan-ainali.html)
+* [Eric Herman](https://publiccode.net/team/eric-herman.html)
+* [Elena Findley-de Regt](https://publiccode.net/team/elena-findley-de-regt.html)
+* [Rasmus Frey](https://os2.eu/bruger/rasmus-frey)
+* [Boris van Hoytema](https://publiccode.net/team/boris-van-hoytema.html)
+
+## Updates from the Foundation
+
+* [The Standard for Public Code has been approved as a Digital Public Good](https://blog.publiccode.net/news/2021/07/28/digital-public-goods.html)
+* [We have started refactoring the introduction](https://github.com/publiccodenet/standard/pull/521)
+
+## Notes
+
+Today's discussion was prompted by two issues about the requirement on [using (plain) English](https://standard.publiccode.net/criteria/understandable-english-first.html).
+We have noted that sometimes, especially in cases coming close to policy actually being executed as code, there is a need to use the language that the policy is written in.
+The issues we have noted this in are:
+
+- Add nuance on where non-English is acceptable [#223](https://github.com/publiccodenet/standard/issues/223)
+- Handling non-English terms - [#308](https://github.com/publiccodenet/standard/issues/308)
+
+We believe this to be important since public policy is core to what we do, it is what makes public code public.
+Likewise the purpose of these requirements -  making codebases reusable - is also core to what we do.
+So it is essential that we find a way to enable collaboration of policy that has been implemented somewhere since it is likely that other pubic organizations will have similar policies.
```

Comment:
```suggestion
So it is essential that we find a way to enable collaboration around already-implemented policy since it is likely that other pubic organizations will have similar policies.
```